### PR TITLE
pwndoc-ng

### PR DESCRIPTION
**Pull request by noraj**
Some changes are safe like `pwndoc/pwndoc` ➡️ `pwndoc-ng/pwndoc-ng` in `*.md` or job or image names in docker files. Some other changes like the container names must be reflected in code or config files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <img src='/docs/_images/logo_text.png' width="200px" />
 
-PwnDoc-ng is a pentest reporting application making it simple and easy to write your findings and generate a customizable Docx report. It is based on original fork of [PwnDoc](https://github.com/pwndoc/pwndoc) work by [yeln4ts](https://github.com/yeln4ts).
+PwnDoc-ng is a pentest reporting application making it simple and easy to write your findings and generate a customizable Docx report. It is based on original fork of [PwnDoc](https://github.com/pwndoc-ng/pwndoc-ng) work by [yeln4ts](https://github.com/yeln4ts).
```
Comment (TheZ3ro) [inline README.md:5]: This should not be renamed since it is a legit link to the original pwndoc

Comment (noraj) [reply README.md:5]: I got too aggressive with match and replace 😅

**Comment by RobertDoc** [comment]
This needs to be tested extensively before merging as it changes quite a lot of config

File: README.md (hunk 1 above)
Comment (Zeecka) [reply README.md]: Nice catch

**Comment by Zeecka** [comment]
LGTM, there is not that much changes.

**Pull request merged**